packages/api/package.json

```diff
@@ -6,10 +6,14 @@
   "types": "./dist/index.d.ts",
   "private": true,
   "scripts": {
-    "build": "tsc -b"
+    "build": "tsc -b",
+    "generateSchema": "typescript-json-schema src/orbitAPI.ts ValidatableSpec -o src/orbitAPISchema.json --noExtraProps --required --ignoreErrors --strictNullChecks"
```

Comment:
Yikes. This is… probably fine, assuming we're generally going to be running a "real" build simultaneously.